assets/src/edit-story/output/test/page.js

```diff
@@ -111,9 +111,10 @@ describe('Page output', () => {
     };
 
     const { container } = render(<PageOutput {...props} />);
-    await expect(queryById(container, 'el-baz')).toBeInTheDocument();
+    const video = queryById(container, 'el-baz-media');
+    await expect(video).toBeInTheDocument();
```

Comment:
Not sure how we can better test that this is actually an `amp-video`. Maybe with a snapshot?

Reply:
Using [Jest DOM's `.toContainHTML`](https://github.com/testing-library/jest-dom#tocontainhtml): `expect(video.parentElement).toContainHTML('<amp-video .... />')` doesn't feel right.

Using [`.toMatchInlineSnapshot`](https://jestjs.io/docs/en/snapshot-testing#inline-snapshots):`expect(video).toMatchInlineSnapshot('<amp-video ... />');` is probably better.